### ISSUE TITLE
Add the ability to load an Avogadro molecule

### ISSUE THREAD

**Issue by cjh1** [OpenChemistry/openchemistrypy#27]
So we can do something like:

```python
import openchemistry as oc
from avogadro import core, io
...
mol = Molecule()
conv = FileFormatManager()
conv.readString(mol, cjson, 'cjson')
oc.load(mol).orbitals.show()
```

**Comment by cjh1** [comment]
The avogadro Molecule instance should be cached.

**Comment by cjh1** [comment]
@cryos

**Comment by alesgenova** [comment]
Fixed by #41
